frontend/dynamic-demo-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# OpenShift Console Demo Plugin
+
+This project emulates a standalone repository hosting a sample
+[dynamic plugin](/frontend/packages/console-dynamic-plugin-sdk/README.md) for OpenShift Console.
+
+It is meant to serve as a reference for Console plugin developers and for testing dynamic plugin
+capabilities via end-to-end tests.
+
+## Local development
+
+1. `yarn build` to build the plugin, generating output to `dist` directory
+2. `yarn http-server` to start an HTTP server hosting the generated assets
+
+```
+Starting up http-server, serving ./dist
+Available on:
+  http://127.0.0.1:9001
+  http://192.168.1.190:9001
+  http://10.40.192.80:9001
+Hit CTRL-C to stop the server
+```
+
+The server runs on port 9001 with caching disabled and CORS enabled. Additional
+[server options](https://github.com/http-party/http-server#available-options) can be passed to
+the script, for example:
+
+```sh
+yarn http-server -a 127.0.0.1 --ssl --cert /path/to/tls.crt --key /path/to/tls.key
+```
+
+## Deployment on cluster
+
+Console dynamic plugins are supposed to be deployed via [OLM operators](https://github.com/operator-framework).
+In case of demo plugin, we just apply a minimal Kubernetes manifest which adds the necessary resources.
```

Comment:
Since we the manifest is containing also a console CRD I would rather use OpenShift.
```suggestion
In case of demo plugin, we just apply a minimal OpenShift manifest which adds the necessary resources.
```